src/components/publishing/image.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+import Fonts from "./fonts"
+
+const Figcaption = styled.div`
+  & > p {
+    ${Fonts.unica("s14")}
+    margin: 0;
+    color: #999;
+  }
+`
+
+const CaptionLink = styled.div`
+  ${Fonts.unica("s14")}
+  margin: 0;
+  border-bottom: 1px solid black;
+  cursor: pointer;
+  display: inline-block;
+`
+
+const CaptionContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 0 10px 0;
+`
+
+const BlockImage = styled.img`
+  display: block;
+`
+
+class Image extends Component<any, null> {
```

Comment:
Btw, in the case of stateless components you can also choose for just [a function](https://facebook.github.io/react/docs/components-and-props.html#functional-and-class-components).